tests/pytests/unit/modules/test_saltutil.py

```diff
@@ -7,8 +9,13 @@
 
 
 @pytest.fixture
-def configure_loader_modules():
-    return {saltutil: {"__opts__": {"file_client": "local"}}}
+def configure_loader_modules(minion_opts):
+    minion_opts["file_client"] = "local"
+    return {
```

Comment:
```suggestion
def configure_loader_modules(minion_opts):
    minion_opts["file_client"] = "local"
    minion_opts["master_uri"] = "tcp://127.0.0.1:4505"
    return {
```
Some test failures are valid.

Reply:
:laughing: This was your code suggestion originally, will commit it

Reply:
yes, but I don't run the tests with my suggestions.
I expect the submitter to do it :grin: